### PR TITLE
Spelling Corrections

### PR DESCRIPTION
**Pull request by CargoVroom**

### CODE REVIEW

File: overrides/contracts/blackout/Blackout_HeavyStatic.json [hunk 1]
```diff
@@ -482,7 +482,7 @@
       "overrideDialogueBucketId": "",
       "dialogueContent": [
         {
-          "words": "Unidentified BattleMech lance. Declare yourself. You are violating a {TEAM_EMP.FactionDef.Demonym} militarised zone",
+          "words": "Unidentified BattleMech lance. Declare yourself. You are violating a {TEAM_EMP.FactionDef.Demonym} militarized zone",
```
Comment (CWolfs) [inline overrides/contracts/blackout/Blackout_HeavyStatic.json:485]: Sorry, I'm British so keeping it the British way :D

File: overrides/contracts/duoduel/DuoDuel_YearlyTussle.json [hunk 2]
```diff
@@ -130,7 +130,7 @@
       "overrideDialogueBucketId": "",
       "dialogueContent": [
         {
-          "words": "You did it! Time to organise the party. We can't let all this go to waste!",
+          "words": "You did it! Time to organize the party. We can't let all this go to waste!",
```
Comment (CWolfs) [inline overrides/contracts/duoduel/DuoDuel_YearlyTussle.json:133]: UK vs US english

**Comment by CargoVroom** [comment]
Done

**Comment by CWolfs** [comment]
Thanks a lot @CargoVroom - you rock!

**Pull request merged**